Add per-operation panels to dashboard

francoisferrand opened the pull request:
- Fix rounding of counts
- Fix last report query
- Add per-operation latency and count
- Fix download/upload chunk size panels

<img width="1609" alt="Screenshot 2022-10-20 at 09 18 21" src="https://user-images.githubusercontent.com/3909027/196882262-26b9048f-af78-4e1f-b52e-ef0e717022cc.png">
<img width="1609" alt="Screenshot 2022-10-20 at 09 19 19" src="https://user-images.githubusercontent.com/3909027/196882253-e3e04351-d54f-4e03-a468-7ad0233281e7.png">
<img width="802" alt="Screenshot 2022-10-21 at 11 47 37" src="https://user-images.githubusercontent.com/3909027/197186960-c0db0fc1-6095-4566-931f-ebbc1bbfb777.png">

Issue: CLDSRV-288

francoisferrand committed:
Fix rounding of counts

Add round() operator in query instead of limiting to 0 decimals, to
allow grafana to display fractional value when there is a "unit" (like
`1.25K`)

Issue: CLDSRV-288

francoisferrand committed:
Fix last report query

Issue: CLDSRV-288

francoisferrand committed:
Add per-operation latency and count

Issue: CLDSRV-288

bert-e commented:
# Hello francoisferrand,



My role is to assist you with the merge of this
pull request. Please type `@bert-e help` to get information
on this process, or consult the [user documentation](
https://bert-e.scality.net/scality/cloudserver/doc/user).





*Status report is not available.*

bert-e commented:
# Waiting for approval



The following approvals are needed before I can proceed with the merge:


* the author


* 2 peers

francoisferrand committed:
Fix chunk upload/download size

The formula is not statistically accurate, but it gives an estimation,
assuming the repartition of object size is somewhat linear.

Issue: CLDSRV-288

francoisferrand committed:
Release 8.6.1

francoisferrand commented:
/approve

bert-e commented:
# In the queue



The changeset has received all authorizations and has been added to the
relevant queue(s). The queue(s) will be merged in the target development
branch(es) as soon as builds have passed.

The changeset will be merged in:


* :heavy_check_mark: `development/8.6`



The following branches will **NOT** be impacted:

* `development/6.4`
* `development/7.10`
* `development/7.4`
* `development/7.70`
* `development/8.4`
* `development/8.5`



There is no action required on your side. You will be notified here once
the changeset has been merged. In the unlikely event that the changeset
fails permanently on the queue, a member of the admin team will
contact you to help resolve the matter.

*IMPORTANT*

Please do not attempt to modify this pull request.

* Any commit you add on the source branch will trigger a new cycle after the
  current queue is merged.
* Any commit you add on one of the integration branches will be **lost**.

If you need this pull request to be removed from the queue, please contact a
member of the admin team now.




*The following options are set:* **approve**

bert-e commented:
# 


I have successfully merged the changeset of this pull request
into targetted development branches:


* :heavy_check_mark: `development/8.6`



The following branches have **NOT** changed:

* `development/6.4`
* `development/7.10`
* `development/7.4`
* `development/7.70`
* `development/8.4`
* `development/8.5`



Please check the status of the associated issue CLDSRV-288.

Goodbye francoisferrand.
